source/documentation/administration_guide/topics/proc-enabling-fips_in_rhv.adoc

```diff
@@ -19,54 +19,21 @@ You can create a FIPS-enabled bare metal machine in RHV 4.4 by either installing
 
 [IMPORTANT]
 ====
-Red Hat recommends installing Red Hat Enterprise Linux 8 with FIPS mode enabled as opposed to enabling FIPS mode later. Enabling FIPS mode during the installation ensures that the system generates all keys with FIPS-approved algorithms and continuous monitoring tests in place.
+The FIPS mode needs to be enabled before RHV host deploy for Clusters with FIPS enabled. Enabling FIPS mode during the installation ensures that the system generates all keys with FIPS-approved algorithms and continuous monitoring tests in place.
 ====
 
 = Enabling FIPS in a self-hosted engine
 
-To create a FIPS-enabled self-hosted engine in RHV version 4.4, link:{URL_downstream_virt_product_docs}{URL_format}installing_{URL_product_virt}_as_a_self-hosted_engine_using_the_command_line/index[deploy the self-hosted engine using the command line]. When the deployment script prompts `Do you want to apply a default OpenSCAP security profile?`, enter `Yes` and follow the on-screen instructions to select the _VPP - Protection Profile for Virtualization 1.0 for Red Hat Enterprise Linux Hypervisor (RHELH)_ profile.
+To create a FIPS-enabled self-hosted engine in RHV version 4.4, link:{URL_downstream_virt_product_docs}{URL_format}installing_{URL_product_virt}_as_a_self-hosted_engine_using_the_command_line/index[deploy the self-hosted engine using the command line]. When the deployment script prompts `Do you want to enable FIPS?`, enter `Yes`.
 
-= Enabling FIPS in RHEL hosts and the standalone Manager
+= Enabling FIPS in RHV hosts and the standalone Manager
 
 For information on switching a provisioned RHEL machine to FIPS mode, see link:{URL_rhel_docs_latest}html/security_hardening/using-the-system-wide-cryptographic-policies_security-hardening#switching-the-system-to-fips-mode_using-the-system-wide-cryptographic-policies[Switching the system to FIPS mode].
 
-For information on installing FIPS-enabled Red Hat Enterprise Linux (RHEL) hosts, see link:{URL_rhel_docs_latest}html/security_hardening/assembly_securing-rhel-during-installation-security-hardening#assembly_installing-a-rhel-8-system-with-fips-mode-enabled_securing-rhel-during-installation}[Installing a RHEL 8 system with FIPS mode enabled].
+For information on installing FIPS-enabled Red Hat Enterprise Linux (RHEL) hosts and Red Hat Virtualization Hosts (RHVH), see link:{URL_rhel_docs_latest}html/security_hardening/assembly_securing-rhel-during-installation-security-hardening#assembly_installing-a-rhel-8-system-with-fips-mode-enabled_securing-rhel-during-installation}[Installing a RHEL 8 system with FIPS mode enabled].
```

Comment:
Please use `{hypervisor-fullname} ({hypervisor-shortname})` instead of `Red Hat Virtualization Hosts (RHVH)`.
Also in the rest of this patch.

Reply:
Done